addons/hr_holidays/static/tests/tours/time_off_allocation_warning_tour.js

```diff
@@ -27,37 +33,49 @@ registry.category("web_tour.tours").add("time_off_allocation_warning_tour", {
         },
         {
             content: "Click to select a leave type",
-            trigger: ".o_field_widget[name='holiday_status_id'] .o-autocomplete--input",
+            trigger: ".o_field_widget[name='holiday_status_id'] input",
             run: "click",
         },
         {
-            trigger: ".o-autocomplete--dropdown-item:nth-child(1) > a",
+            trigger: ".ui-autocomplete > li > a:not(:has(i.fa))",
```

Comment:
```suggestion
            trigger: [".ui-autocomplete > li > a[input=]",](trigger: ".o-autocomplete--dropdown-menu > li > a[id=holiday_status_id_0_0_0]",)
```
what do you think of this trigger ? :)